docs/dev-setup-non-vagrant.md

```diff
@@ -342,7 +342,7 @@ fi
 ./tools/do-destroy-rebuild-test-database
 ./manage.py compilemessages
 sudo ./scripts/lib/install-node
```

Comment:
Made those changes. They are somewhat substantial so someone should take a look at the third commit.